Ugee_G5/index.md

```diff
@@ -0,0 +1,18 @@
+---
+VID: "28bd"
+PID: "0074"
+vendor: Ugee
+product: G5
+image: Ugee_G5
+report_rate: 220
+working_area:
+    width: 9
+    height: 6
+    resolution: 5080
+pen:
+    pressure_levels: 2048
+frame_controls: scroll wheel
+sold_as:
+supported: false
+supported_in:
```

Comment:
`supported` should be `true` now and `supported_in` should have `digimend: ">= 8"`. See e.g. https://raw.githubusercontent.com/DIGImend/tablets/master/Huion_H950P/index.md

Can you set it this way?